api/client.go

```diff
@@ -41,9 +41,9 @@ func checkError(status int, body []byte) error {
 	case http.StatusNotFound:
 		return ErrNotFound
 	case http.StatusBadRequest:
-		return fmt.Errorf("%w: %s", ErrBadRequest, apiError.Message)
+		return fmt.Errorf(apiError.Message)
```

Comment:
I think we'll eventually want to change how `checkError` works, to return an error that allows the caller to retrieve the underlying exit status in all cases. 

Right now we can in some cases, but not in others. And I guess this change makes it so that the caller can no longer determine if it was a 400 status or some other status.

Reply:
I put together #1766 to try and address this.